composer.json

```diff
@@ -78,6 +78,7 @@
         "fidry/makefile": "^0.2.0",
         "helmich/phpunit-json-assert": "^3.0",
         "phpspec/prophecy-phpunit": "^2.0",
+        "phpspec/prophecy": "^1.15",
```

Comment:
could you please explain why is it needed?

Reply:
with prefer-lowest it could get to 1.3.0 which has some issues: https://github.com/infection/infection/actions/runs/4434955792/jobs/7781506076#step:13:71

Reply:
but is it a direct dependency of Infection? If not, probably it's better to add it to `conflicts: {"phpspec/prophecy: "<1.15"}`? does it make sense?

Reply:
it's a dependency from `phpspec/prophecy-phpunit`.

I think correcting transient or indirect dependencies can be done like so. Using `conflict` would make it incompatible as a package, which is not true (it's only a problem for our tests)